Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:latest
 
-MAINTAINER Tess Thyer <tess.thyer@sagebase.org>
+LABEL maintainer="Tess Thyer <tess.thyer@sagebase.org>"
 LABEL base_image="ubuntu:latest"
 LABEL about.summary="Docker image for template dockstore tool"
-LABEL about.license="SPDX:MIT"
+LABEL about.license="SPDX:Apache-2.0"
```

Comment:
Please explain the reason for your proposed license change.

Reply:
I updated the license in the Dockerfile to match the `LICENSE` file included in the repo.